labs/memory_bound/data_packing/validate.cpp

```diff
@@ -6,7 +6,7 @@
 template <typename Received_t, typename Expected_t>
 static bool validate(const char *var_name, Received_t received,
                      Expected_t expected, int first_value, int second_value) {
-  if (received == expected)
+  if (std::abs(float(received - expected)) < 0.001)
```

Comment:
Given that the floating point member we're checking is a double, maybe also go with double instead of float?

Reply:
Yes, I think I will change it to `Received_t`.